apps/test/integration/util/testBlockly.js

```diff
@@ -0,0 +1,65 @@
+// Note: Putting ES6 in this test file breaks the test build, for reasons I
+// haven't figured out yet (bbuchanan).  It's got something to do with require-globify.
+var assert = require('../../util/configuredChai').assert;
+var testUtils = require('../../util/testUtils');
+var testBlockFactory = require('./testBlockFactory');
+
+/** @type {StudioApp} instance reference internal to this module  */
+var studioApp;
+
+/**
+ * Initializes an instance of blockly for testing
+ */
+exports.setupTestBlockly = function () {
+  exports.setupBlocklyFrame();
+  var options = {
+    assetUrl: studioApp.assetUrl
+  };
+  var blocklyAppDiv = document.getElementById('app');
+  Blockly.inject(blocklyAppDiv, options);
+  // TODO (brent)
+  // studioApp.removeEventListeners();
```

Comment:
Not sure there's value in keeping this TODO around at this point (my guess was the desire was that we clear any existing event listeners before starting test?)